### PR TITLE
fix(content): update FIL Token Section

### PR DESCRIPTION
**Pull request by yiannisbot**
This PR applies restructuring to the FIL Token section and adds description to some of the subsections. In particular, the FIL Wallet section is temporarily removed and instead we add the Decay Model and the Token Allocation.

The PR will also update the MultiSig Actor and the pointers to the related repositories.

### CODE REVIEW

File: content/systems/filecoin_token/decay_model.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+title: Decay Model
+bookCollapseSection: true
+weight: 1
+dashboardWeight: 1
+dashboardState: reliable
+dashboardAudit: missing
+dashboardTests: 0
+---
+
+# Decay Model
```
Comment (zixuanzh) [inline content/systems/filecoin_token/decay_model.md:11]: Hybrid Minting? Minting Model?

File: content/systems/filecoin_token/token_allocation.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+| `FIL_StorageMiningAlloc` | 1,100,000,000 FIL | The amount of FIL allocated to storage miners through block rewards, network initialization |
+| `FIL_Vested` | Sum of genesis `MultisigActors.`<br>`AmountUnlocked` | Total amount of FIL that is vested from genesis allocation. | 
+| `FIL_StorageMined` | `RewardActor.`<br>`TotalStoragePowerReward` | The amount of FIL that has been mined by storage miners |
+| `FIL_Locked` | `TotalPledgeCollateral` + `TotalProviderDealCollateral` + `TotalClientDealCollateral` + `TotalPendingDealPayment` + `OtherLockedFunds` | The amount of FIL locked as part of mining, deals, and other mechanisms. |
+| `FIL_CirculatingSupply` | `FIL_Vested` + `FIL_Mined` - `TotalBurntFunds` - `FIL_Locked` | The amount of FIL circulating and tradeable in the economy. The basis for Market Cap calculations. |
+| `TotalBurntFunds` | `BurntFundsActor.`<br>`Balance` | Total FIL burned as part of penalties and on-chain computations. |
+| `TotalPledgeCollateral` | `StoragePowerActor.`<br>`TotalPledgeCollateral` | Total FIL locked as collateral in all miners. |
```
Comment (zixuanzh) [inline content/systems/filecoin_token/token_allocation.md:34]: Total FIL locked as **pledge** collateral in all miners.

**Review by zixuanzh — DISMISSED** [review]
minor suggestions

File: content/systems/filecoin_token/multisig.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+# Multisig Wallet & Actor
+
+
+The Multisig actor is a single actor representing a group of Signers. Signers may be external users, other Multisigs, or even the Multisig itself. There should be a maximum of 256 signers in a multisig wallet. In case more signers are needed, then the multisigs should be combined into a tree. 
+
+The implementation of the Multisig Actor can be found [here](https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/multisig/multisig_actor.go).
+
+The Multisig Actor statuses can be found [here](https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/multisig/multisig_state.go).
```
Comment (hugomrdias) [inline content/systems/filecoin_token/multisig.md:18]: We can use symbols lookup to embed this

**Review by hugomrdias — CHANGES_REQUESTED** [review]

**Review by hugomrdias — APPROVED** [review]

**Pull request merged**